src/Text/Pandoc/Writers/Powerpoint/Presentation.hs

```diff
@@ -29,6 +29,7 @@ module Text.Pandoc.Writers.Powerpoint.Presentation ( documentToPresentation
                                                    , Paragraph(..)
                                                    , ParaElem(..)
                                                    , ParaProps(..)
+                                                   , CodePlacement(..)
```

Comment:
One condern here: on the whole, I want the `Presentation` data type to model the actual powerpoint format as closely as possible, and keep formatting things that aren't in pptx in `Output.hs`. `CodePlacement` isn't something that really has an analogue in PowerPoint,  right?

Would there be any way to keep these changes confined to `Output`?

Reply:
Thank you for getting back on this. I had been pondering about this as PowerPoint has no real concept of code